### DISCUSSION TITLE
What's the best way to test development builds of these packages?

### DISCUSSION THREAD

**Discussion by bgourlie** [nxext/nx-extensions#795] [Q&A]
I've made some changes to the sveltekit extension and would like to test them. I run `npx nx run sveltekit:build` and I see the build artifacts in `dist/` (`npm run build` seems to be broken, for me at least) . What's the best way to test the extension in a separate workspace? Just depend on them using `file:../foo/bar` syntax? Is there a better way?

**Comment by DominikPieper** [comment]
Hey, you can run „npx nx e2e sveltekit-e2e“. That will run the e2e tests and create a workspace under tmp/e2e…/proj in the project folder. Inside that folder a Nx workspace with plugins installed will be created

**Reply by bgourlie** [reply]
Running `npx nx e2e sveltekit-e2e` gives me the following error:

```
Command failed: npm publish -tag latest --access public --json
npm ERR! code ENEEDAUTH
npm ERR! need auth This command requires you to be logged in to http://localhost:4872/
npm ERR! need auth You need to authorize this machine using `npm adduser`
{
  "error": {
    "code": "ENEEDAUTH",
    "summary": "This command requires you to be logged in to http://localhost:4872/",
    "detail": "You need to authorize this machine using `npm adduser`"
  }
}

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/brian/.npm/_logs/2022-08-26T22_51_00_229Z-debug-0.log
```
